README.md

```diff
@@ -15,6 +15,10 @@ How this is done:
 3. Invoke `tsqlust::get_diagnostics_for_tsql(&your_sql)`
 4. Iterate over the resulting `Vec<tsqlust::diagnostics::Diagnostic>` and take action
 
+## Current State of Project
+
+> Currently the project only lexes and parses a query up to a WHERE statement
```

Comment:
`up to a WHERE statement` isn't quite true (it implies there is a linear flow of sql statements).